tests/yast2_gui/yast2_lang.pm

```diff
@@ -20,7 +20,11 @@ use testapi;
 
 sub run {
     my $self = shift;
-    $self->launch_yast2_module_x11('language', match_timeout => 60);
+    # Kill all yast processes which may remain in memory
+    select_console('root-console');
+    script_run('pkill -TERM -e yast2');
+    select_console('x11');
```

Comment:
How about to put that "killing" part in a `if (get_var('YAST2_GUI_TERMINATE_PREVIOUS_INSTANCES'))` to make it optional and trigger it only on demand? If we find out it's more stable this way then put the variable into the test suite for time being and we can run both versions in parallel, one in validation, the other one as "experimental".

Reply:
Sounds good, but let me check first if this is required. So far processes were dead already before. Otherwise, I would even put it in launch_yast2_module_x11 method in the way you've suggested.